twitch-recorder.py

```diff
@@ -149,9 +157,14 @@ def loop_check(self, recorded_path, processed_path):
                 processed_filename = os.path.join(processed_path, filename)
 
                 # start streamlink process
-                subprocess.call(
-                    ["streamlink", "--twitch-disable-ads", "twitch.tv/" + self.username, self.quality,
-                     "-o", recorded_filename])
+                streamlink_args = ["streamlink", "--twitch-disable-ads"]
+                if self.twitch_oauth_token is not None:
+                    logging.info("Use Twitch OAuth token")
+                    streamlink_args.extend(["--twitch-api-header", "Authorization=OAuth " + self.twitch_oauth_token])
+                streamlink_args.extend(["--config", pwd + "/.streamlinkrc"])
+                streamlink_args.extend(["twitch.tv/" + self.username, self.quality, "-o", recorded_filename])
+                logging.info("streamlink args: %s", streamlink_args)
```

Comment:
After dig down twitch dev doc, I finally realize someone already done this 😆 
I forked these changes, but remove this line to avoid logging OAuth token.